src/ccl_class.c

```diff
@@ -489,22 +488,89 @@ void ccl_cosmology_compute_linpower_class(ccl_cosmology* cosmo, int* status) {
     // After this loop lk will contain log(k), lpk_ln will contain log(P_lin), all in Mpc, not Mpc/h units!
     double psout_l;
     s=0;
-    for (int i=0; i<nk; i++) {
-      for (int j = 0; j < na; j++) {
-  //The 2D interpolation routines access the function values pk_{k_ia_j} with the following ordering:
-  //pk_ij = pk[j*N_k + i]
-  //with i = 0,...,N_k-1 and j = 0,...,N_a-1.
-  s |= spectra_pk_at_k_and_z(&ba, &pm, &sp,lk[i],1./aa[j]-1.+1e-10, &psout_l,&ic);
-  lpk_ln[j*nk+i] = log(psout_l);
-      }
-      lk[i] = log(lk[i]);
-    }
+    
+      // If scale-independent mu / Sigma modified gravity is in use 
+      // and mu ! = 0 : get the unnormalized growth factor in MG and for 
+      // corresponding GR case, to rescale CLASS power spectrum
+    if ( fabs(cosmo->params.mu_0)>1e-14){
+	    // Set up another cosmology which is exactly the same as the 
```

Comment:
Here I checked on mu_0 because only mu_0 affects the power spectrum, which is what is being calculated in this file. Similarly, in the case of the checks in ccl_cls.c, only Sigma_0 is checked because only Sigma_0 affects the lensing and magnification tracers directly (dependence on mu0 is automatically inherited from any dependence in P(k)). Does that answer this question?

Reply:
I used mu and sigma for the two cases but what I really meant is the consistency of the if condition 
if fabs(cosmo->params.mu_0)>1e-14
versus 
if (fabs(cosmo->params.mu_0) 

at diffrent places

Reply:
Oh I see. Where are you seeing the second one? (File and line number would be helpful.)

Reply:
It may have been in an older file that I did not stash when I wrote that. If you don't see it don't worry about it.